cvat/apps/engine/serializers.py

```diff
@@ -84,12 +85,19 @@ def update_instance(validated_data, parent_instance):
         else:
             instance['task'] = parent_instance
             logger = slogger.task[parent_instance.id]
-        (db_label, created) = models.Label.objects.get_or_create(name=validated_data['name'],
-            **instance)
-        if created:
-            logger.info("New {} label was created".format(db_label.name))
+        if not validated_data.get('id') is None:
+            db_label = models.Label.objects.get(id=validated_data['id'],
```

Comment:
@ActiveChooN , what if the id isn't valid? How does it handled?

Reply:
Sure, added wrapper with 404 response. Will it be applicable, or just validationError will be enough, what do you think?